code/datums/materials/basemats.dm

```diff
@@ -217,8 +217,9 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	beauty_modifier = -0.01
 	armor_modifiers = list(MELEE = 1.5, BULLET = 1.1, LASER = 0.3, ENERGY = 0.5, BOMB = 1, BIO = 1, RAD = 1, FIRE = 1.1, ACID = 1)
 
-/datum/material/plastic/on_accidental_mat_consumption(mob/living/carbon/M, obj/item/S)
-	M.adjust_disgust(17)
+/datum/material/plastic/on_accidental_mat_consumption(mob/living/carbon/eater, obj/item/food)
```

Comment:
Not necessarily your task but it might be nice to add the amount consumed to this proc and use that to determine the reagent count